projectname/main.py

```diff
@@ -0,0 +1,6 @@
+import pandas as pd
+
+csv_path = "/Users/christophe/Downloads/pcbanking-3.csv"
```

Comment:
Great first step. Can you rename the folder projectname to what ever name you want to give to the project (no space, oneword) ex: `comptable` or `christocount` or what ever:P